### PR TITLE
Renarde documentation in Qute template

### PR DESCRIPTION
**Pull request by angelozerr**
Renarde documentation in Qute template

### CODE REVIEW

File: docs/qute/TemplateSupport.md [hunk 1]
```diff
@@ -94,6 +94,12 @@ A variable can be declared inside a Qute template (with parameter declaration, o
 
 ![Definition for Java data model](images/QuteTemplateDefinitionForObject.gif)
 
+## User tags support
+
+The [User tags](https://quarkus.io/guides/qute-reference#user_tags) are loaded from `src/main/resources/templates/tags` and uses content of user tag to collect parameters from the object part, uses the existing of `{nested-content}` to provide the best support for completion, definition, validation:
```
Comment (JessicaJHee) [inline docs/qute/TemplateSupport.md]: I think we can reword this part to something like: 
... are loaded from `src/main/resources/templates/tags`. The parameters and presence of `{nested-content}` in the user tag declared in the tag template is used to provide support for completion, definition, and validation in the Qute template:

File: docs/renarde/TemplateSupport.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+Qute support provides `completions`, `validation`, `hover`, `definition` for uri/uriabs namespace:
+
+![Renarde Uri support](images/RenardeUriSupport.gif)
+
+## User tags support
+
+For [User tags](https://quarkiverse.github.io/quarkiverse-docs/quarkus-renarde/dev/index.html#_user_tags) please see [User Tags support](../qute/TemplateSupport.md#user-tags-support)
+
+[Renarde tags](https://quarkiverse.github.io/quarkiverse-docs/quarkus-renarde/dev/index.html#_renarde_tags) are loaded from `templates/tags` of the Renarde dependency; it means those tags depend on Renarde version. You can see Renarde `#form` tag in action in the previous demo.
```
Comment (JessicaJHee) [inline docs/renarde/TemplateSupport.md]: ... which means those tags depend on Renarde version.

File: docs/renarde/TemplateSupport.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+### Quick fix
+
+Form renarde is validated and provides some quick fixes:
```
Comment (JessicaJHee) [inline docs/renarde/TemplateSupport.md]: Form Tag in Renarde is validated and support with some quick fixes that will insert either the required inputs forms or all input forms as defined in the method.

Comment (angelozerr) [reply docs/renarde/TemplateSupport.md]: fixed

File: docs/renarde/TemplateSupport.md (hunk 2 above)
Comment (angelozerr) [reply docs/renarde/TemplateSupport.md]: fixed

File: docs/qute/TemplateSupport.md (hunk 1 above)
Comment (angelozerr) [reply docs/qute/TemplateSupport.md]: fixed

**Review by JessicaJHee — APPROVED** [review]
LGTM!

**Review by datho7561 — APPROVED** [review]
Looks good! Thanks, Angelo!

**Pull request merged**